public/includes/da-newsletter.php

```diff
@@ -0,0 +1,134 @@
+<?php
+class DrawAttention_Newsletter {
+	public $parent;
+	public $plugin_directory;
+
+
+	public function __construct( $parent ) {
+		$this->plugin_directory = DrawAttention::get_plugin_url() . '/public/';
+		$this->parent           = $parent;
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_meta_box_assets' ) );
+		add_action( 'admin_footer', array( $this, 'newsletter_modal_dialog' ) );
+		add_action( 'add_meta_boxes', array( $this, 'add_newsletter_widget' ) );
+
+		// Order the meta boxes
+		add_action( 'do_meta_boxes', array( $this, 'set_meta_boxes_position' ) );
+	}
+
+	public function enqueue_meta_box_assets() {
+		wp_enqueue_style( 'custom-meta-box-styles', $this->plugin_directory . '/assets/css/custom-meta-box-styles.css', array(), DrawAttention::VERSION );
+		wp_enqueue_script( 'news-letter-js', $this->plugin_directory . 'assets/js/news-letter.js', array(), DrawAttention::VERSION );
+	}
+
+	public function metabox_newsletter_component() {
+		echo "
+            <div class='news-letter-container hndle ui-sortable-handle'> 
+
+                <div class='content-container'>
+                    <div>
+                        <img src='" . $this->plugin_directory . "/assets/images/news-letter.svg' alt='News Letter Image'>
+                    </div>
+                    <div>
+                        <p> " . __( 'Stay up to date with the latest from Draw Attention', 'draw-attention' ) . "</p>
+                    </div>
+                    <div class='outer-content'>
+                        <p>Subscribe now! Get 20% Coupon</p>
+                        <button id='openModalButton' onclick='openModal()'> <span>" . __( 'SUBSCRIBE', 'draw-attention' ) . "</span> </button>
```

Comment:
Let's change to `Subscribe` and apply `text-transform: uppercase` to achieve the all caps